### ISSUE TITLE
purpur server - mine /list input returns error

### ISSUE THREAD

**Issue by gekigek99** [gekigek99/minecraft-server-hibernation#110]
![image](https://user-images.githubusercontent.com/53654579/126084215-0200d0b6-b974-4b89-a85b-ac8fe8fc495d.png)

**Comment by gekigek99** [comment]
also other commands have the same behavior 

```
mine /time
2021/07/19 00:50:30 GetInput: user input: [mine /time]
2021/07/19 00:50:30 terminal execute: /time     (origin: user input)
[00:50:30 INFO]: Unknown command. Type "/help" for help.
mine /seed
2021/07/19 00:50:50 GetInput: user input: [mine /seed]
2021/07/19 00:50:50 terminal execute: /seed     (origin: user input)
[00:50:50 INFO]: Unknown command. Type "/help" for help.
```

**Comment by gekigek99** [comment]
also `/save-all` command does not work (this is quite and important command to be executed successfully)

```
2021/07/19 00:57:04 saving word before killing the minecraft server process
2021/07/19 00:57:04 terminal execute: /save-all         (origin: sigintMinecraftServerIfOnlineAfterTimeout)
[00:57:04 INFO]: Unknown command. Type "/help" for help.
2021/07/19 00:57:14 send kill signal to minecraft server process since it won't stop normally
2021/07/19 00:57:14 waitForExit: terminal exited
2021/07/19 00:57:14 *** MINECRAFT SERVER IS OFFLINE!
```

**Comment by debian-user-france1** [comment]
> ![image](https://user-images.githubusercontent.com/53654579/126084215-0200d0b6-b974-4b89-a85b-ac8fe8fc495d.png)

Please do not add the / to mine. Simply: mine list - mine time set day.

Please close because it is working solution 100% sure

**Comment by gekigek99** [comment]
> Please close because it is working solution 100% sure

please refrain from this behavior: I am in charge of closing the issue. thank you for the understanding.

> OR IS A PURPUR SERVER DIFFERENT?

please do not scream... apparently yes since a special characters has to be inserted before the `/command`.  
Fortunately when the command is executed server side the `/` can be omitted and the problem does not occur anymore.

**Comment by gekigek99** [comment]
fixed in 13d23da12a7834c94497611be53b98002878a913
